Sources_v3/StreamChat/APIClient/APIClient.swift

```diff
@@ -63,4 +65,52 @@ class APIClient {
             task.resume()
         }
     }
+
+    func uploadFile<Response: Decodable>(
```

Comment:
Should the response be generic or should it always be `FileUploadResponse`? Even if clients use their own CDN we still need to be able to understand the response. So having this hardcoded to `FileUploadResponse` seems reasonable.

Reply:
fixed in 5d1d4587fc27b39cd66541f4ac290cfee037b9a6